dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -192,7 +192,7 @@ def main(args):
     if args.sched_addr:
         client = Client(args.sched_addr)
     else:
-        cluster = Cluster(*cluster_args, **cluster_kwargs)
+        cluster = Cluster(*cluster_args, **cluster_kwargs, rmm_pool_size="20GB")
```

Comment:
We shouldn't enforce some specific size, let's either allow the user to specify a size or use the default (half the GPU's total memory). This will immediately cause problems with 16GB GPUs, which are still very common.

Reply:
That was a mistake .  I'll take that out